srcalc/src/main/webapp/WEB-INF/tags/editVariablePage.tag

```diff
@@ -72,7 +72,7 @@
         <td>
             <form:textarea path="helpText" cols="90" rows="8" />
             <form:errors path="helpText" cssClass="error" />
-            <ul class="helpText">
+            <ul class="postInstructions">
```

Comment:
It refers to instructions that appear after the controls (as opposed to `preInstructions`). I needed to differentiate for styling.